src/main/java/potaton/Potaton.java

```diff
@@ -0,0 +1,299 @@
+                } catch (IndexOutOfBoundsException e) {
+                    // Refactor this later
+                    System.out.println("_____________________________________________\n"
+                            + "Error: The description of a todo cannot be empty." + "\n"
+                            + "_____________________________________________\n"
+                    );
+                }
+                break;
+            case TASK_DEADLINE:
+                addDeadline(line, tasks);
+                break;
+            case TASK_EVENT:
+                addEvent(line, tasks);
+                break;
+            case TASK_DELETE:
+                deleteTask(line, tasks);
+                break;
+            default:
+                printError();
+                break;
+            }
+            line = in.nextLine();
+            command = line.split(" ")[0];
+        }
+    }
+
+    private static void printWelcomeMessage() {
+        String logo = " ______   _____   ________  _______ __________ _______ \n"
+                + "( ____ )(  ___  )\\__   __/(  ___  )\\__   __/(  ___  )( (    /|\n"
+                + "| (    )|| (   ) |   ) (   | (   ) |   ) (   | (   ) ||  \\  ( |\\\n"
+                + "| (____)|| |   | |   | |   | (___) |   | |   | |   | ||   \\ | |\n"
+                + "|  _____)| |   | |   | |   |  ___  |   | |   | |   | || (\\ \\) | \n"
+                + "| (      | |   | |   | |   | (   ) |   | |   | |   | || | \\   | \n"
+                + "| )      | (___) |   | |   | )   ( |   | |   | (___) || )  \\  | \n"
+                + "|/       (_______)   )_(   |/     \\|   )_(   (_______)|/    )_) \n";
+
+        System.out.println("_____________________________________________\n"
+                + logo + "\n"
+                + "Hello! I'm Potaton\n"
+                + "What can I do for you?\n"
+                + "_____________________________________________\n"
+        );
+    }
+
+    private static void addEvent(String line, ArrayList<Task> tasks) {
+        String arg = line.split(" ", 2)[1];
+        String event = arg.split("/at ", 2)[0];
+        String eventDate = arg.split("/at ", 2)[1];
+        Task t = new Events(event, eventDate);
+        tasks.add(t);
+        System.out.println("_____________________________________________\n"
+                + "Got it. I've added this task: \n"
+                +  t + "\n"
+                + "Now you have " + tasks.size() + " tasks in the list." + "\n"
+                + "_____________________________________________\n"
+        );
+    }
+
+    private static void loadEvent(String line, ArrayList<Task> tasks) {
+        String arg = line.split("] ", 2)[1];
+        String event = arg.split(" \\(at: ", 2)[0];
+        String eventDate = arg.split(" \\(at: ", 2)[1].split("\\)")[0];
+        Task t = new Events(event, eventDate);
+        tasks.add(t);
+        System.out.println("_____________________________________________\n"
```

Comment:
You have several `println` statements with line headers like this. How about shortening the print statements by declaring a header variable and reusing it?